coaster/utils.py

```diff
@@ -96,9 +96,9 @@ def uuid1mc_from_datetime(dt):
     >>> # Both timestamps should be very close to each other but not an exact match
     >>> u1.time > u2.time
     True
-    >>> u1.time - u2.time < 3000
+    >>> u1.time - u2.time < 5000
```

Comment:
This fails randomly if the CPU is busy for other reasons. Raised the number to reduce likelihood of failure.